x-pack/plugins/apm/server/routes/service_groups/route.ts

```diff
@@ -75,10 +81,15 @@ const serviceGroupSaveRoute = createApmServerRoute({
   handler: async (resources): Promise<void> => {
     const { context, params } = resources;
     const { start, end, serviceGroupId } = params.query;
-    const savedObjectsClient = (await context.core).savedObjects.client;
+    const {
+      savedObjects: { client: savedObjectsClient },
+      uiSettings: { client: uiSettingsClient },
+    } = await context.core;
+
     const setup = await setupRequest(resources);
```

Comment:
I think you can parallelize the call to `uiSettings` with `setupRequest`.

Reply:
Fixed in 2f050fbbf0fecd0849cccddcce7684e88ef88b15